DEVELOPMENT.md

```diff
@@ -0,0 +1,84 @@
+# Development
+
+This doc explains how to set up a development environment, so you can get started
+contributing to `oam-kubernetes-runtime` or build a PoC (Proof of Concept). 
+
+## Prerequisites
+
+1. Golang version 1.12+
+2. Kubernetes version v1.15+ with kubectl configured
+
+## Build
+
+The functionality of this library can be demonstrated with the following steps:
+
+* Fork and clone this project
+```shell script
+git clone https://github.com/xxxxxx/oam-kubernetes-runtime.git
+```
+
+* Build the library 
+
+```shell
+make submodules 
+
+make
+```
+
+## Make your change
+* Generate and install CRDs to your Kubernetes cluster
+
+```shell
+make generate
+
+kubectl apply -f crds/
+```
+
+* Make your code changes
+
+Make changes and run the following to test it.
+
+If you only change code under [core apis](./apis/core), remember to
+regenerate crds mainifests as the step above.
+
+## Verification
+You can start running OAM Kubernetes runtime to verify your chanes.
```

Comment:
Before this, we need add some prerequisites for example: `minikube `

Reply:
I see you use ContainerizedWorkload as part of workflow, then it means user must install `https://github.com/crossplane/addon-oam-kubernetes-local` as prerequisite

Reply:
1. Kubernetes cluster is a must in part `Prerequisites`. See https://github.com/crossplane/oam-kubernetes-runtime/pull/45/files#diff-f785e9e46b9afd2567df7a658338db17R9 please.
2. Apply manifests in crds will work. See https://github.com/crossplane/oam-kubernetes-runtime/pull/45/files#diff-f785e9e46b9afd2567df7a658338db17R34 please.